app/web/src/features/dashboard/DashboardBanners.tsx

```diff
@@ -1,12 +1,12 @@
-import { Typography } from "@material-ui/core";
+import { Button as MuiButton, Typography } from "@material-ui/core";
```

Comment:
Prefer our `Button` component from `"components/Button"`. It has default variant of contained and you should be able to set colour to default.